src/components/pages/DashboardPage.js

```diff
@@ -49,7 +50,7 @@ export default class DashboardPage extends Component {
           className={
             (this.state.mainStep === 2 ? 'active-class' : '') + ' dashboard mr-2 mt-2 mb-2'
           }
-         disabled={this.state.mainStep != 2}
+         disabled={this.state.mainStep < 2}
```

Comment:
Think about user experience here. If users want to have a look at the other steps they should be able to jump around, especially if they've already completed that step and want a reminder of something. At the moment they have to start from the beginning and go through everything again. You could maybe reduce the opacity of the inactive steps, but still have them clickable? Just a thought.